Cargo.toml

```diff
@@ -31,3 +31,10 @@ members = [
     "radix-engine-constants",
     "utils",
 ]
+
+# Recommended settings for wasmi for best performance
+#  see: https://github.com/paritytech/wasmi/blob/master/CHANGELOG.md#0140---2022-07-26
+[profile.release]
+lto = "fat"
+codegen-units = 1
```

Comment:
Not sure if it will be respected, since this crate is only a library.

May need to inform Network Team to do so.

Reply:
With standard settings we have ~10% reduction in Transaction time benchmark. 
So if 10% is acceptable I can revert it.

Reply:
Let's keep it and ask Network Team to do the same.

Reply:
PR: https://github.com/radixdlt/babylon-node/pull/289